lib/comparison.js

```diff
@@ -277,7 +277,7 @@ helpers.ifEven = function(num, options) {
  */
 
 helpers.ifNth = function(a, b, options) {
-  if (++b % a === 0) {
+  if (b % a === 0) {
```

Comment:
This validates that an array is passed in: https://github.com/assemble/handlebars-helpers/blob/master/lib/comparison.js#L206

These validate the values are numbers in the utils:
https://github.com/assemble/handlebars-helpers/blob/master/lib/comparison.js#L261
https://github.com/assemble/handlebars-helpers/blob/master/lib/comparison.js#L305

I think that checking that both are `utils.isNumber` should be sufficient.